oap-server/server-storage-plugin/storage-iotdb-plugin/src/test/java/org/apache/skywalking/oap/server/storage/plugin/iotdb/base/IoTDBMetricsDAOTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.storage.plugin.iotdb.base;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.skywalking.oap.server.core.analysis.DownSampling;
+import org.apache.skywalking.oap.server.core.analysis.FunctionCategory;
+import org.apache.skywalking.oap.server.core.analysis.manual.instance.InstanceTraffic;
+import org.apache.skywalking.oap.server.core.analysis.metrics.Metrics;
+import org.apache.skywalking.oap.server.core.storage.StorageHashMapBuilder;
+import org.apache.skywalking.oap.server.core.storage.model.ExtraQueryIndex;
+import org.apache.skywalking.oap.server.core.storage.model.Model;
+import org.apache.skywalking.oap.server.core.storage.model.ModelColumn;
+import org.apache.skywalking.oap.server.storage.plugin.iotdb.IoTDBClient;
+import org.apache.skywalking.oap.server.storage.plugin.iotdb.IoTDBStorageConfig;
+import org.apache.skywalking.oap.server.storage.plugin.iotdb.IoTDBTableMetaInfo;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.apache.skywalking.oap.server.storage.plugin.iotdb.IoTDBClientTest.retrieval;
+
+public class IoTDBMetricsDAOTest {
```

Comment:
This kind of tests belong to integration test, refer to other existing storage tests about how to test.

Reply:
I need this method to build `org.apache.skywalking.oap.server.core.storage.model`, and add it to `IoTDBTableMetaInfo.TABLE_META_INFOS`. When those query tests run, they need to get information from `IoTDBTableMetaInfo`.

Reply:
Other storage tests just test their tool class. However I tests those query methods directly.

Reply:
Could we only test these DAOs in e2e rather than in Maven IT? It reduces resource costs.

Reply:
SkyWalking doesn't trust maven IT(not personal or relate to this PR, just general idea), SkyWalking is too complex, it is always better to have real OAP up and running, then verify logic.
IT in maven covers the API level check.